php/admin/class-license.php

```diff
@@ -120,17 +128,18 @@ public function activate_license( $key ) {
 		$response = wp_remote_post(
 			$this->store_url,
 			array(
-				'timeout'   => 15,
+				'timeout'   => 10,
```

Comment:
This seemed a little hight, compared to the default of `5`. But maybe it's appropriate.